registry/proxy/proxymetrics.go

```diff
@@ -21,29 +37,43 @@ type proxyMetricsCollector struct {
 }
 
 // BlobPull tracks metrics about blobs pulled into the cache
-func (pmc *proxyMetricsCollector) BlobPull(bytesPulled uint64) {
+func (pmc *proxyMetricsCollector) BlobPull(bytes uint64) {
 	atomic.AddUint64(&pmc.blobMetrics.Misses, 1)
-	atomic.AddUint64(&pmc.blobMetrics.BytesPulled, bytesPulled)
+	atomic.AddUint64(&pmc.blobMetrics.BytesPulled, bytes)
+
+	misses.WithValues("blob").Inc(1)
+	bytesPulled.WithValues("blob").Inc(float64(bytes))
 }
 
 // BlobPush tracks metrics about blobs pushed to clients
-func (pmc *proxyMetricsCollector) BlobPush(bytesPushed uint64) {
+func (pmc *proxyMetricsCollector) BlobPush(bytes uint64) {
```

Comment:
Wondering if `bytesPushed` was used here to prevent potential collisions with stdLib's `bytes` package (although it looks like that's not currently imported here).